src/components/graphCard/graphCardChartTooltip.js

```diff
@@ -45,31 +45,32 @@ const GraphCardChartTooltip = ({
       };
 
       if (itemsByKey[key]?.chartType === ChartTypeVariant.threshold) {
-        let thresholdStringValue = itemsByKey[key]?.data.y ?? t('curiosity-graph.label_noData');
+        let thresholdStringValue = itemsByKey[key]?.data.y ?? t('curiosity-graph.label', { context: 'noData' });
 
-        if (itemsByKey[key]?.data.hasInfinite) {
+        // ToDo: Remove hasInfinite once deprecated graphCard components are removed
+        if (itemsByKey[key]?.data.hasInfinite || itemsByKey[key]?.data.hasInfiniteQuantity) {
           if (tempDataFacet.color) {
             thresholdStringValue = (
               <ChartIcon
                 symbol="infinity"
                 fill="#ffffff"
-                title={t('curiosity-graph.label_threshold', { context: 'infinite' })}
+                title={t('curiosity-graph.label', { context: ['threshold', 'infinite'] })}
               />
             );
           } else {
-            thresholdStringValue = t('curiosity-graph.label_threshold', { context: 'infinite' });
+            thresholdStringValue = t('curiosity-graph.label', { context: ['threshold', 'infinite'] });
```

Comment:
required:
this display string can be consolidated with the previous condition and a slight mod to the locale string

```
if (itemsByKey[key]?.data.hasInfinite || itemsByKey[key]?.data.hasInfiniteQuantity) {
   thresholdStringValue = t('curiosity-graph.label', { context: ['threshold', 'infinite'] }, [
      tempDataFacet.color && (
         <ChartIcon
            symbol="infinity"
            fill="#ffffff"
            title={t('curiosity-graph.label', { context: ['threshold', 'infinite'] })}
            />
         )
      ...
```

and 
```
"label_threshold_infinite": "<0>unlimited</0>",
```